actionview/lib/action_view/helpers/asset_url_helper.rb

```diff
@@ -158,6 +167,14 @@ def asset_path(source, options = {})
       end
       alias_method :path_to_asset, :asset_path # aliased to avoid conflicts with an asset_path named route
 
+      # Computes the path to an asset in the public folder.
+      # This uses +asset_path+ and skips any asset lookups by assuming the asset is in the
```

Comment:
On these docs, should it be `+path_to_asset+`? Same question for the ones below.